target/linux/bmips/dts/bcm6328.dtsi

```diff
@@ -0,0 +1,450 @@
+				pins = "hsspi_cs1";
+			};
+
+			pinctrl_usb_port1_device: usb_port1_device {
+				function = "usb_device_port";
+				pins = "usb_port1";
+			};
+
+			pinctrl_usb_port1_host: usb_port1_host {
+				function = "usb_host_port";
+				pins = "usb_port1";
+			};
+		};
+
+		uart0: serial@10000100 {
+			compatible = "brcm,bcm6345-uart";
+			reg = <0x10000100 0x18>;
+
+			interrupt-parent = <&periph_intc>;
+			interrupts = <BCM6328_IRQ_UART0>;
+
+			clocks = <&periph_osc>;
+
+			status = "disabled";
+		};
+
+		uart1: serial@10000120 {
+			compatible = "brcm,bcm6345-uart";
+			reg = <0x10000120 0x18>;
+
+			interrupt-parent = <&periph_intc>;
+			interrupts = <BCM6328_IRQ_UART1>;
+
+			clocks = <&periph_osc>;
+
+			status = "disabled";
+		};
+
+		nflash: nand@10000200 {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			compatible = "brcm,nand-bcm6368",
+				     "brcm,brcmnand-v2.2",
+				     "brcm,brcmnand";
+			reg = <0x10000200 0x180>,
+			      <0x10000400 0x200>,
+			      <0x10000070 0x10>;
+			reg-names = "nand",
+				    "nand-cache",
+				    "nand-int-base";
+
+			interrupt-parent = <&periph_intc>;
+			interrupts = <BCM6328_IRQ_NAND>;
+
+			status = "disabled";
+		};
+
+		leds: led-controller@10000800 {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			compatible = "brcm,bcm6328-leds";
+			reg = <0x10000800 0x24>;
+
+			status = "disabled";
+		};
+
+		hsspi: spi@10001000 {
+			compatible = "brcm,bcm6328-hsspi";
+			reg = <0x10001000 0x600>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			interrupt-parent = <&periph_intc>;
+			interrupts = <BCM6328_IRQ_HSSPI>;
+
+			clocks = <&periph_clk BCM6328_CLK_HSSPI>,
+				 <&hsspi_osc>;
+			clock-names = "hsspi",
+				      "pll";
+
+			resets = <&periph_rst BCM6328_RST_SPI>;
+
+			status = "disabled";
+		};
+
+		serdes_cntl: syscon@10001800 {
+			compatible = "syscon";
+			reg = <0x10001800 0x4>;
+			native-endian;
+		};
+
+		periph_pwr: power-controller@10001848 {
+			compatible = "brcm,bcm6328-power-controller";
+			reg = <0x10001848 0x4>;
+
+			#power-domain-cells = <1>;
+		};
+
+		ehci: usb@10002500 {
+			compatible = "brcm,bcm6328-ehci", "generic-ehci";
+			reg = <0x10002500 0x100>;
+			big-endian;
+			ignore-oc;
+
+			interrupt-parent = <&periph_intc>;
+			interrupts = <BCM6328_IRQ_EHCI>;
+
+			phys = <&usbh>;
+
+			status = "disabled";
+		};
+
+		ohci: usb@10002600 {
+			compatible = "brcm,bcm6328-ohci", "generic-ohci";
+			reg = <0x10002600 0x100>;
+			big-endian;
+			no-big-frame-no;
+
+			interrupt-parent = <&periph_intc>;
+			interrupts = <BCM6328_IRQ_OHCI>;
+
+			phys = <&usbh>;
+
+			status = "disabled";
+		};
+
+		usbh: usb-phy@10002700 {
+			compatible = "brcm,bcm6328-usbh";
+			reg = <0x10002700 0x38>;
+
+			#phy-cells = <0>;
+
+			clocks = <&periph_clk BCM6328_CLK_USBH>;
+			clock-names = "usbh";
+
+			resets = <&periph_rst BCM6328_RST_USBH>;
+			power-domains = <&periph_pwr BCM6328_PWR_USBH>;
+
+			status = "disabled";
+		};
+
+		enet: ethernet@10e00000 {
```

Comment:
Any reason for the abbreviated names? I'd just have used "ethernet" like on ramips ...